addons/website_sale/controllers/website.py

```diff
@@ -54,11 +54,11 @@ def _login_redirect(self, uid, redirect=None):
         return super()._login_redirect(uid, redirect=redirect)
 
     @route()
-    def autocomplete(self, search_type=None, term=None, order=None, limit=5, max_nb_chars=999, options=None):
+    def autocomplete(self, search_type=None, term=None, order=None, limit=5, offset=0, max_nb_chars=999, options=None):
         options = options or {}
         if 'display_currency' not in options:
             options['display_currency'] = request.website.currency_id
-        return super().autocomplete(search_type, term, order, limit, max_nb_chars, options)
+        return super().autocomplete(search_type, term, order, limit, offset=offset, max_nb_chars=max_nb_chars, options=options)
```

Comment:
```suggestion
        return super().autocomplete(search_type, term, order, limit, offset, max_nb_chars, options=options)
```
Won't this work? (as we are modifying only options.)